internal/store/networkpolicy_test.go

```diff
@@ -30,8 +30,8 @@ func TestNetworkPolicyStore(t *testing.T) {
 	metav1StartTime := metav1.Unix(int64(startTime), 0)
 
 	const metadata = `
-		# HELP kube_verticalpodautoscaler_labels Kubernetes labels converted to Prometheus labels.
-		# TYPE kube_verticalpodautoscaler_labels gauge
+		# HELP kube_networkpolicy_labels Kubernetes labels converted to Prometheus labels.
+		# TYPE kube_networkpolicy_labels gauge
```

Comment:
is it just me or that is a bit sus?

Ah I see, it is unused. Normally you would have that on top of tests were you have headers in commons and want to simplify the header verificaiton as in: https://github.com/kubernetes/kube-state-metrics/blob/80e67d01a20f1a16147d8acb4d474a46ccc4971b/internal/store/certificatesigningrequest_test.go

let's remove it for now and fix it in a follow up?

Reply:
I found it while I was looking for vpa/verticalpodautoscaler and thought this will improve it at least a bit. I can drop it.

Reply:
it doesn't seem to be used right, so I am with either add the logic to use it in this PR as in https://github.com/kubernetes/kube-state-metrics/blob/80e67d01a20f1a16147d8acb4d474a46ccc4971b/internal/store/certificatesigningrequest_test.go#L231 or drop it and do it in a follow-up

Reply:
Sounds good, I dropped it for now.